crates/cxx-qt-lib/Cargo.toml

```diff
@@ -53,6 +59,8 @@ time = ["dep:time"]
 url = ["dep:url"]
 serde = ["dep:serde"]
 uuid = ["dep:uuid"]
+imagev24 = ["dep:imagev24"]
+imagev25 = ["dep:imagev25"]
```

Comment:
this does continue to raise the fun in the Rust ecosystem due to the orphan rules around which crate should implement things and how many different features we should support for the types. Would still be nice if developers could easily add their own implementations easily

Reply:
Yeah, this part is a bit annoying.
But I guess this is the best way forward for now?

Reply:
yup was more an observation